types/usb/index.d.ts

```diff
@@ -382,6 +382,12 @@ export interface Endpoint extends EventEmitter {
 
   /** Object with fields from the endpoint descriptor -- see libusb documentation or USB spec. */
   descriptor: EndpointDescriptor;
+
+  /**
+   * Clear the halt/stall condition for this endpoint.
+   * @param callback
```

Comment:
`@param` is generally redundant.

Reply:
Perhaps, but it was to match the style of the rest of the jsdoc comments